api/python/provisioner/commands/upgrade/get_swupgrade_info.py

```diff
@@ -177,15 +127,19 @@ def run(self, iso_path: str = None,
             Cortx repo metadata
 
         """
+        run_args = GetSWUpgradeInfoArgs(iso_path, release)
```

Comment:
Why do we need this if we have 
```
  _run_args_type = GetSWUpgradeInfoArgs
```
?

Reply:
`_run_args_types` is used as a declaration of vars but is not called by generic logic so no converters and validators are applied, e.g. relative paths leaves as is at it failed in some cases during testing (gpg could not find a file)